src/main/resources/application.properties

```diff
@@ -66,7 +66,7 @@ camel.component.servlet.mapping.context-path=/api/xavier/*
 # WIR parallel creation
 parallel.wir=true
 
-insights.properties=yearOverYearGrowthRatePercentage,percentageOfHypervisorsMigratedOnYear1,percentageOfHypervisorsMigratedOnYear2,percentageOfHypervisorsMigratedOnYear3, reportName, reportDescription,file
+insights.properties=yearOverYearGrowthRatePercentage,percentageOfHypervisorsMigratedOnYear1,percentageOfHypervisorsMigratedOnYear2,percentageOfHypervisorsMigratedOnYear3,reportName,reportDescription
```

Comment:
Is this change related to the multi-json?

Reply:
This is to avoid overriding the property in the deployment properties in OCP
Now there is a : insights.properties , with a value without the "file" in the xavier-integration deployment conf in CI.